spec/queries/district_analytics_query_spec.rb

```diff
@@ -198,8 +222,8 @@
 
     before do
       Timecop.travel(three_months_back) do
-        create(:blood_pressure, patient: patients.first, facility: facility_2)
-        create(:blood_pressure, patient: patients.second, facility: facility_2)
+        create(:blood_pressure, :with_encounter, patient: patients.first, facility: facility_2)
```

Comment:
The new followup scopes we're using rely on encounters being present for the BPs ([ref](https://github.com/simpledotorg/simple-server/blob/reports-activity-tables/app/models/facility.rb#L103)). I think this requirement is redundant, but I've chosen to update the test setup for now.

I don't know if it's safe to assume that all BPs have an encounter because #famouslastwords. I'd like to update the `Facility.hypertension_follow_ups_by_period` scope to operation on `Patient.stuff` instead of `patients.stuff` instead.